independent-projects/qute/core/src/main/java/io/quarkus/qute/ReflectionValueResolver.java

```diff
@@ -202,4 +210,26 @@ static String decapitalize(String name) {
         return new String(chars);
     }
 
+    class CachedAccessorResolver implements ValueResolver {
```

Comment:
I'm looking at this quickly so maybe I'm wrong..need `static`?

Reply:
Ah, yes, it could be `static`. I originally wanted to delegate some logic to the `ReflectionValueResolver` but forgot to update the code afterwards.